nss-day-cohort-46/holiday-road-unnamed-riddle
# Cr save to json

# Description

Save button active when user makes 3 selections. 
Save selections to db.json as object with selection names as keys.
`
let itinerary = {

      parkName: partkName,

      eateryName: eateryName,

      attractionName: attractionName
}
`

## Type of change

Please delete options that are not relevant.


# Testing Instructions

`git fetch --all`
`git checkout cr-save-to-json`
`serve`

In other terminal

`json-server -p 8088 -w db.json`

View itinerary at

` http://localhost:8088/itineraries`


- By default, save button should be disabled.
- User makes 3 selections and button should turn green.
- If user changes one or more selections to the default value, button is disabled again.
- If user makes 3 selections and saves, button turns blue and selections are saved to db.json
- If user makes 3 selections, saves, and selects a default option, button is disabled.
- If user makes 3 selections, saves, and makes another valid selection, button turns from blue to green.


# Checklist:

- [X ] I have performed a self-review of my own code
- [] I have added test instructions that prove my fix is effective or that my feature works
